### PR TITLE
feat: update bg chapter 8

### PR DESCRIPTION
**Pull request by Popespice**
## Summary
Updates to chapter 8 of the MAR BG

### Location
docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md

Discord username (if different from GitHub): Popespice

### CODE REVIEW

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
+<link rel="stylesheet" href="/stylesheets/bg.css">
 # After Landing and Taxi to Gate
 
-This guide will explain the correct procedures after we have landed and vacated the runway and then taxiing to the designated gate.
+This guide will explain the correct procedures after we have landed and vacated the runway, and will also cover taxiing to the designated gate.
 
 !!! warning "Disclaimer"
-    The level of detail in this guide is meant to get a FlyByWire A320neo beginner from the runway to the designated destination gate.
+    The level of detail in this guide is meant to get a FlyByWire A380X beginner from the runway to the designated destination gate.
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]:
```suggestion
    The level of detail in this guide is meant to get a Airbus A380 beginner from the runway to the designated destination gate.
```

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 ---
 
 ## Prerequisites
 
 Aircraft has vacated the runway completely and has been brought to a stop on the taxiway as per previous chapters.
 
-![Runway vacated](../assets/beginner-guide/after-landing/rwy-vacated.png "Runway vacated"){ loading=lazy }
+![Runway vacated](../assets/beginner-guide/after-landing/rwy-vacated.png "Runway vacated"){ loading=lazy } <p style="color:yellow; font-size:18px;">TODO: update/screenshot</p>
 
 [Download FlyByWire Checklist](../assets/sop/A32NX%20Documentation/FBW%20A32NX%20Checklist.pdf){ .md-button }
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]:
```suggestion
[Download FlyByWire Checklist](../assets/sop/A32NX%20Documentation/FBW%20A32NX%20Checklist.pdf){ .md-button }
<p style="color:yellow; font-size:18px;">TODO: update link</p>
```

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 3]
```diff
@@ -51,45 +51,65 @@ ATC Tower will usually hand us off to ATC Ground, and they will give us taxi ins
 
     ![EDDM ATC frequencies](../assets/beginner-guide/after-landing/EDDM-frequency-chart.png "EDDM ATC frequencies"){ loading=lazy }
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]: needs to be changed to EDDF

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ This guide will cover these phases:
 - ATC has been informed that we vacated the runway.
 
 !!!info "Simulation vs. Real Life"
-    In real life, the A320 will have two pilots who can actually do things in parallel. Talking to ATC, taxiing the aircraft and doing the after landing tasks.
+    In real life, the A380 will have two pilots who can actually do things in parallel. Talking to ATC, taxiing the aircraft and doing the after landing tasks.
 
     In the simulation we are typically alone, so it is perfectly fine to stop once we have fully vacated the runway and do these things one after the other.
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]: the below should go into an admon

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 5]
```diff
@@ -51,45 +51,65 @@ ATC Tower will usually hand us off to ATC Ground, and they will give us taxi ins
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]: Same as in the review for chapter 9 - please use the A380 SOP as your guide not the "old" A32NX bg pages.
Remove all this floating text and use the SOP lines as much as you can. 
Use the "How and Where" admon below the sections. 

I will not repeat this comment throughout the whole page as it is the same for most of the sections. 

Example:
image

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 6]
```diff
@@ -51,45 +51,65 @@ ATC Tower will usually hand us off to ATC Ground, and they will give us taxi ins
...
 
-    ![RMP Panel](../assets/beginner-guide/after-landing/RMP-Panel.png "RMP Panel"){loading=lazy }
+    ![RMP Panel](../assets/beginner-guide/after-landing/RMP-Panel.png "RMP Panel"){loading=lazy } <p style="color:yellow; font-size:18px;">TODO: update/screenshot</p>
 
 - Disarm the `Speed Brake` lever (Ground Spoilers) by pushing down on the lever.
 
-    ![Spoiler armed and disarmed](../assets/beginner-guide/after-landing/Spoiler.png "Spoiler armed and disarmed"){loading=lazy }
+    ![Spoiler armed and disarmed](../assets/beginner-guide/after-landing/Spoiler.png "Spoiler armed and disarmed"){loading=lazy } <p style="color:yellow; font-size:18px;">TODO: update/screenshot</p>
+
+`GROUND SPOILERS ...................................................... OFF`<br/>
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]: image
```suggestion
`GROUND SPOILERS ................................................... DISARM`<br/>
```

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 7]
```diff
@@ -51,45 +51,65 @@ ATC Tower will usually hand us off to ATC Ground, and they will give us taxi ins
...
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]: in the A380 SOP the next item would be "Flaps", then APU

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md [hunk 8]
```diff
@@ -30,104 +29,165 @@ This guide will cover these phases:
...
+    to. Online ATC controllers understand that the after-landing-tasks do take some time.
+
+    If you did not get any frequencies from ATC, you can use the airport charts to find the correct frequencies for EDDM.
+
+    ![EDDM ATC frequencies](../../a32nx/assets/beginner-guide/after-landing/EDDM-frequency-chart.png "EDDM ATC frequencies"){ loading=lazy }
+    <p style="color:yellow; font-size:18px;">TODO: change to EDDF</p>
+`GROUND SPOILERS ................................................... DISARM`<br/>
+`FLAPS ............................................................ RETRACT`<br/>
+`AUXILLIARY POWER UNIT START ........................................... ON`<br/>
+??? note "Starting the APOU"
```
Comment (2hwk) [inline docs/pilots-corner/a380x/a380x-beginner-guide/08_after-landing.md]:
```suggestion
??? note "Starting the APU"
```

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Pull request merged**